config/environments/production.rb

```diff
@@ -36,7 +36,9 @@
   # config.action_dispatch.x_sendfile_header = 'X-Accel-Redirect' # for NGINX
 
   # Store uploaded files on the local file system (see config/storage.yml for options).
-  config.active_storage.service = :local
+  # config.active_storage.service = :local
```

Comment:
Just remove this commented-out line, it's cruft. We know how to change if we want.